data/gettingstarted.html

```diff
@@ -1281,7 +1325,11 @@ <h3>&zwnj;Window shortcuts - general&zwnj;</h3>
                   <i class="material-icons">&#xE145;</i>
                   <b class="btn btn-xs disabled">S</b>
                 </td>
-                <td>&zwnj;Call Window Shuffler gui (window tiling grid).&zwnj;</td>
+                <td>&zwnj;Call Window Shuffler gui (window tiling grid).&zwnj;
+<ul>
+  <li><a onclick="cmd('link?https://www.youtube.com/watch?v=Ii4n8H4XEzc&feature=youtu.be')">&nbsp;Window Shuffler Demonstration&nbsp;</a></li>
```

Comment:
Would be better to have a button here as per https://github.com/UbuntuBudgie/budgie-welcome/blob/master/data/store.html#L30

Reply:
Added, but can't figure out how to keep the list formatting without adding weird characters below  the line. Removed list formatting, and the button is in a silly location, maybe?